src/CMakeLists.txt

```diff
@@ -69,7 +69,10 @@ endif()
 # Global flags and include directories
 # When generating a dtrace header file, symbols containing dollar-signs are created This file needs to be compiled as well.
 # Hence, the `-Wno-dollar-in-identifier-extension` flag is required.
-add_compile_options(-std=c++17 -pthread -Wall -Wextra -pedantic -Werror -Wno-unused-parameter -Wno-dollar-in-identifier-extension -Wno-unknown-pragmas)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
```

Comment:
Especially with the new line below, the comment above `-Wno` does no longer match the code beneath it.

Reply:
Ok. Is there a better place to put the `CXX_STANDARD` and `CXX_STANDARD_REQUIRED` declarations? At the top of the file or even in the root CMakeLists.txt file after line 2?
https://github.com/hyrise/hyrise/blob/6ce8213580fa87253ed628574f346c605e06a0b3/CMakeLists.txt#L1-L5

Reply:
The place itself is not bad. You just need to move the last two lines of the comment to where they belong.

Reply:
Ok 👍